doc/classes/String.xml

```diff
@@ -590,7 +615,7 @@
 			<argument index="1" name="from" type="int" default="-1">
 			</argument>
 			<description>
-				Perform a search for a substring, but start from the end of the string instead of the beginning. Also search case-insensitive.
+				Performs a search for a substring, but starts from the end of the string instead of the beginning. Also searches case-insensitive.
```

Comment:
Also searches case-insensitive. -> "insensitive" is an adjective, but we need an adverb here. However, "searches case-insensitively" sounds really weird. How about:

Search is case-insensitive.